datacenter/ucp/2.1/guides/admin/upgrade/release-notes.md

```diff
@@ -10,6 +10,59 @@ known issues for the latest UCP version.
 You can then use [the upgrade instructions](index.md), to
 upgrade your installation to the latest release.
 
+
+## Version 2.1.1
+
+(13 Mar 2017)
+
+**New features**
+
+* Core
+  * Administrators can now configure the frequency with which UCP polls metrics.
+  Use `docker service update --env-add METRICS_SCRAPE_INTERVAL=2h ucp-agent`,
+  and the frequency can be in s/m/h/d.
+  * Support for syncing users and teams from multiple LDAP servers/domains
+  (e.g. a separate server to use for `dc=domain2,dc=example,dc=com`)
+  * Support for limiting the number of maximum concurrent login sessions any
+  user may have
+
```

Comment:
/cc @wsong we should talk about `METRICS_DISK_USAGE_INTERVAL`

This is both a new feature and a bug fix.

The bug fix is that we now only gather disk metrics once every 2 hours (by default)

The feature is adding the ability to tune that.  The model is ~identical to the scrape interval above, just set the env `METRICS_DISK_USAGE_INTERVAL=12h` as an example to gather it twice a day.